test/libsolidity/smtCheckerTests/file_level/import.sol

```diff
@@ -0,0 +1,28 @@
+==== Source: A ====
+struct S { uint x; }
+function set(S storage a, uint v) { a.x = v; }
+
+==== Source: B ====
+import "A";
+import "A" as A;
+contract C {
+	A.S data;
+	function f(uint v) internal returns (uint one, uint two) {
+		A.set(data, v);
+		one = data.x;
+		set(data, v + 1);
+		two = data.x;
+	}
+	function g() public {
+		(uint x, uint y) = f(7);
+		assert(x == 7); // should hold but the SMTChecker doesn't know that
```

Comment:
Is this because we are passing `data` in function `f` to `A.set` as a storage pointer, or because of some other reason?

Reply:
In this case storage pointer